### ISSUE TITLE
Clang format issue

### ISSUE THREAD

**Issue by zyd2001** [microsoft/vscode-cpptools#5403]
**Type: LanguageService**

**Describe the bug**
- OS and Version: Windows 10 1909
- VS Code Version: 1.44.2
- C/C++ Extension Version: 0.27.1
- Other extensions you installed (and if the issue persists after disabling them): none
- Does this issue involve using SSH remote to run the extension on a remote machine?: no
- A clear and concise description of what the bug is, including information about the workspace (i.e. is the workspace a single project or multiple projects, size of the project, etc). The clang-format comes with this extension has a bug with pointer alignment. Newer clang-format has corrected the bug

**Steps to reproduce**

format following code
```cpp
class C{
    C operator&(const C&)const;
    C operator*(const C&)const;
    C operator+(const C&)const;};
```
with `.clang-format`
```
BasedOnStyle: Microsoft
AccessModifierOffset: '-4'
PointerAlignment: Middle
TabWidth: '4'
```
The result of `operator&(const C &) const` and `operator*` isn't correct.
```cpp
class C
{
    C operator&(const C &)const;
    C operator*(const C &)const;
    C operator+(const C &) const;
};
```

**Expected behavior**

should looks like this
```cpp
class C
{
    C operator&(const C &) const;
    C operator*(const C &) const;
    C operator+(const C &) const;
};
```
I test with the latest version of clang-format. The bug is fixed. The version comes with this extension has this bug.

<details>
  <summary><strong>Logs</strong></summary>

```
Insert logs here.
```
</details>

**Screenshots**

**Additional context**

**Comment by sean-mcmanus** [comment]
Sounds like resolving https://github.com/microsoft/vscode-cpptools/issues/5194 would fix this issue.

**Comment by sean-mcmanus** [comment]
This should work now with https://github.com/microsoft/vscode-cpptools/releases/tag/0.29.0-insiders .

**Comment by sean-mcmanus** [comment]
Actually, it doesn't work with the vsix from the marketplace, which still uses clang-format 9. We'll fix that for our next release. The offline vsix should work though.

Should be working again with https://github.com/microsoft/vscode-cpptools/releases/tag/0.30.0-insiders (and should be fixed with 0.30.0), but be aware that it may be falling back to the vcFormat implementation if no .clang-format exists.
